### PR TITLE
Set up weekly updates

### PR DESCRIPTION
**Pull request by brittktrinh**
Code for https://github.com/ParkingReformNetwork/organization-dashboard/issues/45

### CODE REVIEW

File: .github/workflows/weekly.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  workflow_dispatch:
+  schedule:
+    - cron: "0 21 * * SUN"
+
+jobs:
+  weekly:
+    if: github.repository_owner == 'ParkingReformNetwork'
+    runs-on: ubuntu-latest
+    env:
+      ENV_VAR_1: ${{ secrets.INFLUXDB_API_TOKEN }}
```
Comment (Eric-Arellano) [inline .github/workflows/weekly.yml:13]: It should be like this:

```suggestion
      INFLUXDB_API_TOKEN: ${{ secrets.INFLUXDB_API_TOKEN }}
```

Ditto on the the other ones.

File: .github/workflows/weekly.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      - name: Install dependencies
+        run: npm ci
+      - name: Run progam
+        run: npm run start -- --services --map-projects
+      - name: Set up environment variables
+        run: |
+          echo $ENV_VAR_1
+          echo $ENV_VAR_2
+          echo $ENV_VAR_3
+          echo $ENV_VAR_4
```
Comment (Eric-Arellano) [inline .github/workflows/weekly.yml:34]: You'll want to remove this

File: tests/utils.test.ts [hunk 3]
```diff
@@ -2,6 +2,8 @@ import { expect, test } from "@playwright/test";
 
 import { runProcess, convertDateToTimeStampS } from "../src/utils";
 
+import mapProjects from "../../mandates-map";
```
Comment (Eric-Arellano) [inline tests/utils.test.ts:5]: How come this change?

**Review by Eric-Arellano — COMMENTED** [review]
Thanks!

File: tests/utils.test.ts (hunk 3 above)
Comment (brittktrinh) [reply tests/utils.test.ts]: I forgot exactly how this happened, I think I was messing around with another branch and accidentally committed this. I'm removing this to match the main branch!

File: tests/utils.test.ts [hunk 4]
```diff
@@ -27,6 +27,14 @@ test.describe("runProcess", () => {
   });
 });
 
+test("maps_mandates", async () => {
```
Comment (Eric-Arellano) [inline tests/utils.test.ts:30]: Ditto on this change. What is it from?

Tip when opening a pull request: review your own code. Check what code has changed and make sure it is all intentional and the best code you can write.

**Review by Eric-Arellano — APPROVED** [review]

**Pull request merged**